src/pool/srv_metrics.c

```diff
@@ -10,312 +10,122 @@
 #include <abt.h>
 #include <gurt/telemetry_producer.h>
 
-/*
- * Parent directory in metrics tree for all the per-pool metric directories
- */
-#define POOL_METRICS_DIR_FMT	"pool/current/%s"
-
 /*
  * Size in bytes of each per-pool metric directory.
  */
-#define POOL_METRICS_DIR_BYTES	(8 * 1024)
-
-/**
- * Global pool metrics
- */
-struct pool_metrics ds_global_pool_metrics;
-
-/**
- * Per-pool metrics
- */
-static d_list_t		per_pool_metrics;
-static ABT_mutex	per_pool_lock;
-
-struct per_pool_entry {
-	struct ds_pool_metrics	metrics;
-	d_list_t		link;
-};
+#define POOL_METRICS_DIR_BYTES	(96 * 1024)
```

Comment:
I had to bump this significantly since i was getting a lot of allocation failures. There might be some optimization we could consider in the format to reduce this size, but that's probably good enough for how.